csharp/ql/src/semmle/code/csharp/dataflow/SignAnalysis.qll

```diff
@@ -244,7 +250,9 @@ private predicate unknownSign(Expr e) {
     or
     e instanceof ArrayAccess and e.getType() instanceof NumericOrCharType
     or
-    e instanceof MethodAccess and e.getType() instanceof NumericOrCharType
+    e instanceof PropertyAccess and e.getType() instanceof NumericOrCharType
+    or
+    e instanceof MethodCall and e.getType() instanceof NumericOrCharType
```

Comment:
@hvitved FYI, it looks like this was the main problem: `MethodAccess` vs `MethodCall`. On `var x = SomeCall();`, I was not getting back the fact that `x` could be pos, neg or zero, and therefore had no viable sign for `x` at all.

I'll add more tests, and clean up this PR before a proper review.